deployments/tekton.go

```diff
@@ -342,15 +342,15 @@ func applyTektonStaging(c *kubernetes.Cluster, ui *ui.UI) (string, error) {
 	}
 	defer os.Remove(tmpFilePath)
 
-	return helpers.Kubectl(fmt.Sprintf("apply -n carrier-workloads --filename %s", tmpFilePath))
+	return helpers.Kubectl(fmt.Sprintf("apply -n fuseml-workloads --filename %s", tmpFilePath))
 }
 
 func createTektonIngress(c *kubernetes.Cluster, subdomain string) error {
 	_, err := c.Kubectl.ExtensionsV1beta1().Ingresses("tekton-pipelines").Create(
 		context.Background(),
 		// TODO: Switch to networking v1 when we don't care about <1.18 clusters
 		// Like this (which has been reverted):
-		// https://github.com/SUSE/carrier/commit/7721d610fdf27a79be980af522783671d3ffc198
+		// https://github.com/SUSE/fuseml/commit/7721d610fdf27a79be980af522783671d3ffc198
```

Comment:
I would keep this as is. The new URL doesn't work.

Reply:
OK, I will move this back but in the end we will probably remove all traefik related stuff